configure.ac

```diff
@@ -74,6 +76,16 @@ AC_ARG_ENABLE([completion-polling],
     [AC_HELP_STRING([--enable-completion-polling],
                     [Enable polling in quiet, fence, and local completion operations (default:disabled)])])
 
+AC_ARG_ENABLE([hwloc-enforce-single-socket],
+    [AC_HELP_STRING([--enable-hwloc-enforce-single-socket],
+                    [Repin process if assigned affinity to cores across multiple sockets (default:disabled)])])
```

Comment:
Slightly confusing wording... perhaps something like this?:
```suggestion
                    [Repin process to ensure it has affinity to cores within a single socket (default:disabled)])])
```
That might be more correct too... If this option is enabled, do we check for funky pinning first?  My understanding is that the code "repins" regardless of the initial pinning.

Reply:
That is correct - repinning will occur regardless of initial pinning if SOS has been configured with single socket or single NUMA node options